app/models/redirection.rb

```diff
@@ -7,6 +7,8 @@ class Redirection < ActiveRecord::Base
   validates :url, presence: true
 
   scope :in_ring_order, -> { order(next_id: :desc) }
+  scope :latest_first, -> { order(created_at: :desc) }
+  scope :most_recent, -> { order(created_at: :desc).first }
```

Comment:
I'm not sure this is a scope -- it doesn't return a chainable `ActiveRecord::Relation`, it returns a specific record. How about:

```ruby
class Redirection
  def most_recent
    self.class.latest_first.first
  end
end
```